tests/stability/test_stability.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import uuid
+
+import coiled
+import dask
+from dask.distributed import Client
+
+SOFTWARE = f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}"
+
+
+def test_shuffle_stability():
+    with coiled.Cluster(
+        software=SOFTWARE,
+        name="coiled-stability-tests_" + str(uuid.uuid4()),
```

Comment:
All of the tests will run on coiled, so we can probably remove the word coiled. Just a detail.